intro1.py

```diff
@@ -0,0 +1,16 @@
+# Write a script that reads your name from the console and prints it reversed in upper-case letters.
+
+user_name = raw_input ('Please enter your name:' )
```

Comment:
Be careful with the differences between python 2 and 3. raw_input was removed in python 3: http://sebastianraschka.com/Articles/2014_python_2_3_key_diff.html#parsing-user-inputs-via-input